lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+require_relative './dish.rb'
+require_relative './order.rb'
+
+class Menu
+  attr_reader :menu_list
+
+  def initialize
+    @dishes = []
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
```

Comment:
This is a good way of adding dishes to the order. Should it be linked to the dishes class?

Reply:
I think was confused. Does the require relative mean you don't need to initialize a new instance of Dish?